vms_activity/views/vms_order_line_view.xml

```diff
@@ -4,6 +4,9 @@
         <field name="model">vms.order.line</field>
         <field name="inherit_id" ref="vms.vms_order_line_form"/>
         <field name="arch" type="xml">
+            <xpath expr="//field[@name='order_id']" position="after">
+                <field modifiers="{}" name="priority" widget="priority"/>
```

Comment:
@AlexisReza  do you need the modifiers attribute??

Reply:
Nope, i just copy it from another module, and i did not realize